### ISSUE TITLE
problem on the accuracy

### ISSUE THREAD

**Issue by kaishxu** [nickvosk/sigir2020-query-resolution#1]
Hello,

I am replicating the experiment on your dataset, the BERT-based classifier is as follow. However, I can not achieve the result in the paper with F1 score 68.7 and 78.5 (Mine is 57.8). Since the model is quite straightforward, is there anything else about the model wrong with your version?
```python
def squeeze(x):
    x = tf.squeeze(x, axis=[-1])
    return x

def build_bert():
    bert_model = load_trained_model_from_checkpoint(config_path, checkpoint_path, trainable=True, seq_len=None)

    x1_in = Input(shape=(None,))
    x2_in = Input(shape=(None,))
    masked = Input(shape=(None,))

    x = bert_model([x1_in, x2_in])
    x = Dropout(0.1)(x)
    x = Dense(1, activation='sigmoid')(x)
    x = Lambda(squeeze)(x)
    p = Multiply()([x, masked])

    model = Model([x1_in, x2_in, masked], p)
    model.compile(loss='binary_crossentropy',
            optimizer=Adam(lr=3e-6, clipvalue=1),
            metrics=[F1_score()])
    return model
```
Could you please help me figure it out?

Thanks a lot!
